overlord/snapstate/handlers_link_test.go

```diff
@@ -1271,6 +1271,10 @@ func (s *linkSnapSuite) setMockKernelRemodelCtx(c *C, oldKernel, newKernel strin
 }
 
 func (s *linkSnapSuite) TestMaybeUndoRemodelBootChangesUnrelatedAppDoesNothing(c *C) {
+	runner := state.NewTaskRunner(s.state)
+	mgr, err := snapstate.Manager(s.state, runner)
```

Comment:
we have a manager in s.snapmgr, any reason not to use that?

Reply:
Right, I missed that, thanks!